changelog.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## What's New (09/12/2017)
+
+### Resolved issues/Bug fixes
+-	Better camelcased suggestions
+-	Better handling of empty operationIds in OpenAPI specs
+-	Case insensitive checks for top level resource properties
+-	Better messaging for tracked resources related rules
+
+### New validation rules
+-	ArraySchemaMustHaveItems – A schema of array type must always contain an items property. without it, AutoRest will fail to generate an SDK. Documentation [link](https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#L88)
+-	LicenseHeaderMustNotBeSpecified - x-ms-code-generation-settings must not have the license section specified in the OpenAPI documents since each generated SDK can have a different licensing header. This information must be provided either from the command line or the configuration file when actually generating the sdk. Documentation [link](https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#R2065)
+-	PostOperationIdContainsUrlVerb - A POST operation's operationId should contain the verb indicated at the end of the corresponding url. Documentation [link](https://github.com/Azure/azure-rest-api-specs/blob/8ac36944d0ab7f6d07daf077a99bdae873d5e3f6/documentation/openapi-authoring-automated-guidelines.md#R2066)
+
```

Comment:
are we missing a couple of changes/rules which @sarangan12 worked on?

Reply:
Yep, still waiting on them.